substratevm/src/com.oracle.graal.pointsto/src/com/oracle/graal/pointsto/meta/AnalysisType.java

```diff
@@ -823,7 +823,7 @@ public final boolean isInitialized() {
     @Override
     public void initialize() {
         if (!wrapped.isInitialized()) {
-            throw GraalError.shouldNotReachHere("Classes can only be initialized using methods in ClassInitializationFeature: " + toClassName());
+            universe.hostVM.initializeType(this);
```

Comment:
I don't think this is a valid change. `HostVM.initializeType` is not about class initialization, it is a more general initialization method. And it is already called for every `AnalysisType` as part of the `initializationTask`.